packages/desktop-client/src/components/schedules/index.js

```diff
@@ -102,3 +101,5 @@ export default function Schedules() {
     </Page>
   );
 }
+
+export default connect(null, actions)(Schedules);
```

Comment:
Didn't you merge a PR to remove all `connect`s just a couple of days ago?

Reply:
Good catch! I think I originally wrote this code before doing that PR. Updated to `useActions`